src/Tools/dotnet-trace/CommandLine/Commands/ConvertCommand.cs

```diff
@@ -12,21 +14,22 @@ internal static class ConvertCommandHandler
     {
         public static int ConvertFile(IConsole console, FileInfo inputFilename, TraceFileFormat format, FileInfo output)
         {
-            if ((int)format <= 0)
+            if (!Enum.IsDefined(format))
             {
-                Console.Error.WriteLine("--format is required.");
+                string options = string.Join(", ", Enum.GetNames(typeof(TraceFileFormat)).Where(x => x != TraceFileFormat.NetTrace.ToString()));
+                console.Error.WriteLine($"Please specify a valid option for the --format. Valid options are: {options}.");
```

Comment:
It would be nice if the help text for the command and this error were consistent about what options are supported. Currently the help text says NetTrace is a valid option and this error says it is not a valid option. I don't know what would be involved in overriding the help text to remove NetTrace from the list, but the alternative of handling the NetTrace option successfully by doing a file copy doesn't appear that hard.

Reply:
https://github.com/dotnet/diagnostics/pull/4514#discussion_r1492835451

we might need some more checks to verify the `--format NetTrace` path is indeed performing `NetTrace` -> `NetTrace`. Even after that, it feels silly to have the convert tool copy a NetTrace to a different file name. I would prefer removing the format option/updating the docs to not include `NetTrace` over making the tool perform an action that users can accomplish with `cp`